tests/test_data/ad_lookup/single_return.json

```diff
@@ -0,0 +1,85 @@
+{
+  "dataVersion": "string",
+  "centres": [
+    {
+      "locationID": "1001",
+      "locationName": "Sheffield Central Library",
+      "welshLanguageLocationName": "Welsh Sheffield Central Library",
+      "distanceInMiles": 1.3,
+      "latitude": "53.380582",
+      "longitude": "-1.466986",
+      "address": "Surrey Street, Sheffield",
+      "welshLanguageAddress": "Welsh Street, Sheffield",
+      "postcode": "S1 1XZ",
+      "phone": "01142734712",
+      "email": "test@email.com",
+      "website": "www.helloworld.com",
+      "description": "string",
+      "welshLanguageDescription": "string",
+      "openingHours": [
+        {
+          "Monday": [
+            {
+              "start": 1030,
+              "finish": 1715
+            }
+          ],
+          "Tuesday": [
+            {
+              "start": 1000,
+              "finish": 1700
+            }
+          ],
+          "Wednesday": [
+            {
+              "start": 1000,
+              "finish": 1700
+            }
+          ],
+          "Thursday": [
+            {
+              "start": 1000,
+              "finish": 1700
+            }
+          ],
+          "Friday": [
+            {
+              "start": 1000,
+              "finish": 1700
+            }
+          ],
+          "Saturday": [
+            {
+              "start": 1000,
+              "finish": 1300
+            }
+          ],
+          "Census Saturday": [
+            {
+              "start": 1000,
+              "finish": 1600
+            }
+          ],
+          "Census Sunday": [
+            {
+              "start": 1000,
+              "finish": 1600
+            }
+          ]
+        }
```

Comment:
consider adding bank holidays too

Reply:
Single return now includes values for all days